app/api/models/karyotype.py

```diff
@@ -0,0 +1,27 @@
+"""
+.. See the NOTICE file distributed with this work for additional information
+   regarding copyright ownership.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+"""
+
+from pydantic import BaseModel, Field, AliasChoices, validator
+from typing import List, Dict, Any, ClassVar
+
+
+class Karyotype(BaseModel):
+    name: str
+    type: str = Field(default="chromosome")
+    length: int
+    is_circular: bool = Field(default=False)
```

Comment:
Could you please hard-code somewhere that if the genome is from E. coli, its chromosome is circular?